bag-of-holding.cabal

```diff
@@ -28,7 +28,6 @@ common commons
   build-depends:
     , aeson           ^>=1.4
     , base            >=4.12 && <5
-    , chainweb        ^>=1.0
```

Comment:
This reduces (unstripped) binary size from 66 to 44 MB and allows many transitive dependencies to be dropped.